tests/test_processing_stack.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import os
+import time
+
+from pytest import mark
+
+APPLICATION_TAG_KEY = "ApplicationName"
+APPLICATION_TAG_VAL = "geospatial-data-lake"
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# TODO: get resources from correct AWS stack (DEPLOY_ENV), if multiple stacks are deployed to
+# single AWS account
+ENV = os.environ.get("DEPLOY_ENV", "dev")
+
+
+@mark.infrastructure
+def test_should_return_healthy_compute_environment(batch_client):
+    datalake_compute_environment = {}
+
+    response = batch_client.describe_compute_environments()
+    for compute_environment in response["computeEnvironments"]:
+        if (
+            APPLICATION_TAG_KEY in compute_environment["tags"].keys()
+            and compute_environment["tags"][APPLICATION_TAG_KEY] == APPLICATION_TAG_VAL
+        ):
```

Comment:
This could be simplified to `if compute_environment["tags"].get(APPLICATION_TAG_KEY, None) == APPLICATION_TAG_VAL:`.

Reply:
Done in 24267c3eca463c99e4193102d1bf67c3c6854d92 . Thanks for this suggestion.